### DISCUSSION TITLE
Translation of functions, or moving the coordinates around

### DISCUSSION THREAD

**Discussion by wrharthur** [lululxvi/deepxde#1657] [Q&A]
Dear Dr. Lu and the community,

DeepXDE is magnificant and I am trying to use it to solve a 2D system invovling coupled inequality constraints besides the differential equations part:
$f_1(x-c_1,y) \geq f_2(x,y)$
and
$f_2(x-c_2,y) \geq f_2(x,y)$
where $c_1$ and $c_2$ are two parameters, and $f_1(x,y)$ and $f_2(x,y)$ are functions to be solved.

Is it possible to translate $f_1$ and $f_2$ so that we can compare their values with the other one valued at somewhere else(i.e. x-c)? Or, is there any other way that we can "extract" values of functions at different coordinates?

Thank you!

**Comment by lululxvi** [comment]
It is doable. You need to prepare the parewise point locations by translation.
